### ISSUE TITLE
Freelss 1,22

### ISSUE THREAD

**Issue by earlofkent** [hairu/freelss#34]
I have installed the 1,22 freelss and still have a problem with it not allowing me to do and update "0 Error" also when i go into change the camera to 5mp on the reapi it defaults to mock camera. i have to reboot to restart the sytem camera comes up as 1.2mp change it to 5mp and the sam result mock camera.
I am able to save all other changes ??
need some help please.

**Comment by spilz87** [comment]
Same issue, did you solve it ? How ?
Thanks

**Comment by earlofkent** [comment]
Hi i fimally reinstalled the software with the upgraded version and all worked

**Comment by spilz87** [comment]
Thanks for your reply, what do you mean by upgraded version ? 1.22 is the last one, no ?

**Comment by stefi01** [comment]
i use v1.26

**Comment by davewmc** [comment]
Where did you find v1.26?

**Comment by stefi01** [comment]
have to buy a license from http://store.murobo.com/atlas-3d-software-license-only/

**Comment by stefi01** [comment]
it is updated and has more functions, pain in the ass to upgrade so if it dont work at first then email them and they will send a link to the direct download

**Comment by davewmc** [comment]
Thank you, I was looking on here for a place to buy the license and couldn't find it.

**Comment by stefi01** [comment]
glad to have helped :)

**Comment by 3boysdad** [comment]
Realize this is an old thread @stefi01 - how long did it take for the murobo store to get back to you?  i was an original kickstarter supporter and used their contact me form, along with my registration key, to ask about upgrading from the free 1.22 version that i installed on my Rpi 3.  this was five days ago with no response.

**Comment by davewmc** [comment]
I had to email him and ask him how long it was going to take. He replied
fairly quickly.

**Comment by stefi01** [comment]
> Realize this is an old thread @stefi01 - how long did it take for the murobo store to get back to you? i was an original kickstarter supporter and used their contact me form, along with my registration key, to ask about upgrading from the free 1.22 version that i installed on my Rpi 3. this was five days ago with no response.

heres mine [https://drive.google.com/file/d/1gbajzcSTFD95-vFw4-p9aXrJOdxhJzHf/view?usp=sharing](url)

**Comment by 3boysdad** [comment]
Thanks @stefi01 - i got it d/l

very kind of you.  i'll cherish this image until my Pi goes dark.  presume this for Rp3?

**Comment by stefi01** [comment]
pi 2

**Comment by 3boysdad** [comment]
So...finally found a solution.  For those stumble upon this thread and wonder...

"i bought a license, how do i get  latest version on my Rpi 3 b(+)?"

It's honestly rather simple. the root of the problem is the certificate on www.murobo.com server expired back in 2018.  This prevents the download from occurring.  Frankly I was skeptical of this working when I thought of it, but I had nothing really to lose.

After you setup a fresh disk image, setup your Rpi 3b (+), and install the FreeLSS as outlined in the instructions within the readme perform the following.

1. SSH out to your Pi.
2. Enter this command : **sudo systemctl stop systemd-timesyncd**.  This will only pause the time sync process, when you reboot it will start up again.
3. Then this command : **sudo date -s '2017-10-02 15:00:00'** 
4. With your browser navigate to your browser where FreeLSS is running.
5. Under settings, click the "Check for Update" button.
6. The updated version will download from the murobo.com site.  Once the download has completed the patch will be applied. 
 Then the Rpi will reboot.
7. After restarting, SSH back to your Rpi, follow the "Running FreeLSS" instructions.  You'll have set it up to run automatically after the update.

**Comment by stefi01** [comment]
which raspberry pi o/s or still the last version of stretch?

**Comment by 3boysdad** [comment]
The lastest version from the Raspberry Pi site.  Nothing special frankly.  I was generally surprised it simply worked.

**Comment by stefi01** [comment]
for me on r pi 3b i try this, says update successful, restarts and v1.22 is listed as installed, shame my version that works wont work on pi3.

**Comment by 3boysdad** [comment]
Deleted my comment - looked a bit more.  The tar file downloaded successfully, it attempted to patch, but it's getting a segmentation error.  Which likely suggests the code has been compiled for for a different OS or CPU.  Going to continue experimenting.

**Comment by stefi01** [comment]
the code you got from me is for the final version of stretch and works on pi 2 only

**Comment by 3boysdad** [comment]
Yep it that's what I found out this morning. The key to get the download working is overriding the pi date/time so the cert expiration doesn't get in the way.

**Comment by babouline** [comment]
hi guys, i got the same problem with brand new pi2 ... can someone share me his image ? tx !
